packages/rs-dpp/src/data_contract/document_type/index/mod.rs

```diff
@@ -28,23 +28,6 @@ pub struct Index {
 }
 
 impl Index {
-    /// Check to see if two objects are conflicting
-    pub fn objects_are_conflicting(&self, object1: &ValueMap, object2: &ValueMap) -> bool {
```

Comment:
No, we shouldn't remove this... this could be useful for outside libraries even.